tests/test_browser.py

```diff
@@ -816,22 +816,14 @@ def test_sdl_key(self):
           open(os.path.join(self.get_dir(), 'pre.js'), 'w').write('''
             function keydown(c) {
              %s
-              //out('push keydown');
-              var event = document.createEvent("KeyboardEvent");
-              event.initKeyEvent("keydown", true, true, window,
-                                 0, 0, 0, 0,
-                                 c, c);
+              var event = new KeyboardEvent("keydown", { 'keyCode': c, 'charCode': c, 'view': window, 'bubbles': true, 'cancelable': true });
```

Comment:
Is there a reason to set cancelable and bubbles on all these events?

Reply:
One or two tests do need those, but most don't - I kept this the same as the tests previously had, though, to change as little as possible in this PR.